.idea/.idea.NUnit2To3SyntaxConverter/.idea/encodings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'm sorry, it seems my .gitignore does not ignore all .idea files. Would you kindly update the gitignore and remove these .idea files?